.travis.yml

```diff
@@ -1,18 +0,0 @@
-language: ruby
-sudo: false
-
-before_install:
-  - gem install bundler -v "$(grep -A 1 BUNDLED.WITH Gemfile.lock | tail -n 1)"
-
-before_script:
-  - "RAILS_ENV=test bundle exec rake db:create"
-  - "RAILS_ENV=test bundle exec rake db:schema:load"
-
-notifications:
-  email:
-    recipients:
-      - tab@zeus.ugent.be
-    on_success: never
-    on_failure: change
-  slack:
-    secure: G2bPwlS5GR3EkCqdeeE5usS0k/JKEErShIED0M1+/IKhTf6Yn7r4S4KZARndIG6B2y4nZ5EF0ONbdqZkOTwj6QaPv+2NNqmh2CK9Q6BqEN/5fyx5zfVUjCJNRnXPVWuyhooYaKbaDO23KtwvsW0ZpFjS+pZzTZNEWI4UOo2ojIEW1jZfFHYJ0U/W6HX8j65VwWP0/I7od7XBs7UgNavndtJV8sQyRXCvlGfkkYe8MgxG/6QdCV9z46i+IbfPkvETMFIhrx0CpohV00TsVOmuPWpqbi86YV5EzdSWImdc3Dyei+SG6xSi3mfaocEjEvb3btHYBYRwCzBU+ObYLHmz7FdG3Vqzyhqf9c8hzuAAUduXYX9W1sEHi6nfwQqquuibXhptpyS/Np7wqSGhqdbdmTbgWpdAbTEwg9i9ZGgV+YpGhYMypA2yGA/7x+akoHJoKEL7qGsbncifjJlOqYoAVVVj67PFIszyDQiGBlMFddn233oqT+3odyiJGWLaXZY+7Ew1sE1BX5GpFxO71EwMZMxhm0FDtittpkK/uhX05d4DvPElxUmIHWEf5UMvQxjrqpSuYjXLM7xEVn2zktZ1Skk6X4wGU6bSV2Jy+JpzVvkVBXzTf9JMZSMbDKwypwcUjXBn+7/frv5QXkCJrKsRIPm8Ox69YNsDuNz/EWpcQsI=
```

Comment:
Are these still used? If they are we can migrate them over, but this might also be an opportunity to clean these up?

Reply:
@Silox Since you added these ^^